Lombiq.BaseTheme/Models/ZoneDescriptor.cs

```diff
@@ -47,7 +47,7 @@ public ZoneDescriptor(string zoneName = null, string elementName = null, bool wr
         var id = ZoneName.ToCamelCase();
         var layoutClassName = string.IsNullOrEmpty(parent)
             ? "layout" + ZoneName
-            : FormattableString.Invariant($"layout{parent}__{id}");
+            : string.Create(CultureInfo.InvariantCulture, $"layout{parent}__{id}");
```

Comment:
This seems to be very verbose. Can't we have some shorter helper method in HL?

For example:
```c#
public static class StringCreator
{
    public static string Invariant(this DefaultInterpolatedStringHandler value) =>
        string.Create(CultureInfo.InvariantCulture, ref value);
}
```

Then this line would become:

```suggestion
            : StringCreator.Invariant($"layout{parent}__{id}");
```

Reply:
Or:
```suggestion
            : InvariantString.New($"layout{parent}__{id}");
```

Reply:
I looked into this but didn't try `DefaultInterpolatedStringHandler` (since you can't simply pass around `FormattableString`, simple approaches won't work), thanks.